lib/py/src/ext/types.cpp

```diff
@@ -20,6 +20,8 @@
 #include "ext/types.h"
 #include "ext/protocol.h"
 
+#include <iostream>
```

Comment:
Mm I believe this was a holdover from testing & using print statements.  It should be safe to remove.  Thanks for catching.